packages/orchestration/src/cosmos-api.ts

```diff
@@ -335,9 +401,28 @@ export interface IcaAccount extends IcaAccountMethods {
   getPort: () => Port;
 }
 
+/**
+ * Full-featured ICA account methods.
```

Comment:
help me understand what "full-featured' means here. Why have variable amount of features?